pyipn/universe.py

```diff
@@ -243,8 +243,12 @@ def from_dict(cls, d, locked=False):
 
             pointing = Pointing(value["pointing"]["ra"],
                                 value["pointing"]["dec"])
-
-            det = Detector(location, pointing, eff_area, name)
+            
+            if 'GALILEO' in name:
```

Comment:
hi, I think again, it would be better to do this outside the class. So you would calculate the background of Galileo and then put this either into the yaml file or when instantiating the python class.